src/client/index.js

```diff
@@ -34,20 +35,22 @@ const examples = [
 
 $(window).on('load', () => {
 
+$('#circuit-icon').append(circuitIconSvg);
```

Comment:
Doing this in code feels ugly, IMHO. Why not get the SVG inlined in HTML or CSS? There are solutions for both of these possibilities (see, for example, https://stackoverflow.com/questions/69408473/how-do-you-selectively-inline-an-svg-in-css-with-webpack-5).

And while you're at it, why not inline the logo, too?

Reply:
I tried to inline this SVG using method described in this stackoverflow post – unfortunately it didn't work for me, and I couldn't figure out why. I've managed to render SVG as a background image in .scss of course. But couldn't force webpack to inline it.

I don't like the solution with appending the SVG to the div, but I give up and don't have any idea how it might get inlined the other way.

Reply:
I refactored it once again and successfully inline'ed the circuit button icon and also the DigitalJS logo in this commit: 03d5fb06668d1f3e4627c00f7b020dd1912b19db